perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -187,6 +187,15 @@ def SetupProxy(self):
     if commands:
       self.RemoteCommand(";".join(commands))
 
+  def UpdateHosts(self):
```

Comment:
Please rename the method to be more descriptive or make the method more generic. e.g. UpdateHostsFille(self, ip, hostname)